src/ViewLayer/DebugDisplay/HomePage/HomePageUi/HomePageUi.cpp

```diff
@@ -21,9 +21,9 @@ QPushButton& HomePageUi::controlButton()
     return *ui_->controlButton;
 }
 
-QPushButton& HomePageUi::motorFaultButton()
+QPushButton& HomePageUi::faultsButton()
```

Comment:
Should just be `faultButton`, or the ui's should be `faultsButton`, but this function should have the same name as the component it's returning.